rules/windows/builtin/system/microsoft_windows_windows_update_client/win_system_susp_system_update_error.yml

```diff
@@ -1,7 +1,10 @@
 title: Windows Update Error
 id: 13cfeb75-9e33-4d04-b0f7-ab8faaa95a59
 status: stable
-description: Detects Windows update errors including installation failures and connection issues. Defenders should observe this in case critical update KB aren't installed.
+description: |
+    Detects Windows update errors including installation failures and connection issues. Defenders should observe this in case critical update KB aren't installed.
```

Comment:
```suggestion
    Detects Windows update errors including installation failures and connection issues. Defenders should observe this in case critical update KBs aren't installed.
```